includes/RequestWiki/RequestWikiRequestViewer.php

```diff
@@ -240,6 +247,31 @@ public function getFormDescriptor(
 				$context->getOutput()->addHTML( Html::errorBox( $wmError ) );
 			}
 
+			if ( $permissionManager->userHasRight( $userR, 'createwiki' ) ) {
+				if ( $permissionManager->userHasRight( $userR, 'createwiki-deleterequest' ) ) {
+					$viewerCwDeleteLevel = 1;
+				} else {
+					$viewerCwDeleteLevel = ( $permissionManager->userHasRight( $userR, 'createwiki-suppressrequest' )
+						? 2 : 0 );
+				}
+				$requesterCount = $wm->getUserRequestCount( $request->requester, $viewerCwDeleteLevel, '*' );
+				if ( $requesterCount >= $this->config->get( 'CreateWikiRequestCountWarnThreshold' ) ) {
+					$context->getOutput()->addHTML(
+						Html::warningBox(
+							Html::rawElement(
+								'p',
+								[],
+								wfMessage(
+									'createwiki-error-requestcountwarn',
+									$requesterCount,
+									$request->requester->getName()
+								)
+							)->parse(),
```

Comment:
HTML::rawElement is a string, perhaps you meant to apply this to wfMessage instead? eg:
```php
wfMessage(
	'createwiki-error-requestcountwarn',
	$requesterCount,
	$request->requester->getName()
)->parse()
```